<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Informations</class>
 <widget class="QMainWindow" name="Informations">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="ongletsInfos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ongletDerniereInfo">
     <attribute name="title">
      <string>Dernière Information</string>
     </attribute>
     <widget class="QTextEdit" name="derniereInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>775</width>
        <height>465</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ongletAnciennesInfos">
     <attribute name="title">
      <string>Informations plus anciennes</string>
     </attribute>
     <widget class="QTextEdit" name="anciennesInfos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>775</width>
        <height>465</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="afficherInfosDemarrage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les informations au démarrage</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
